docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -166,6 +166,14 @@ Dates listed indicate the start date of the maintenance cycle.
      - End Of Life
      - Control Node Python
      - Target Python / PowerShell
+   * - `2.19`
```

Comment:
This should be a link to https://github.com/ansible/ansible/blob/stable-2.19/changelogs/CHANGELOG-v2.19.rst.